src/components/conjunction/ConjunctionObjectData.tsx

```diff
@@ -0,0 +1,50 @@
+import dayjs from 'dayjs';
+import { useTranslations } from 'next-intl';
+
+import type { TypeSatelliteOut } from '@/__generated__/data-contracts';
+import { FORMAT_DATE_TIME } from '@/libs/Dayjs';
+
+import RichText from '../RichText';
+import { AdditionalInformationsTable } from '../satellite/tables/AdditionalInformationsTable';
+import { BaseInformationsTable } from '../satellite/tables/BaseInformationsTable';
+import { LicenseInformationsTable } from '../satellite/tables/LicenseInformationsTable';
+import { OrbitalInformationsTable } from '../satellite/tables/OrbitalInformationsTable';
+
+type ConjunctionObjectDataProps = {
+  primaryObject: TypeSatelliteOut;
+  secondaryObject: TypeSatelliteOut | null;
+};
+
+const ConjunctionObjectData = ({ primaryObject, secondaryObject }: ConjunctionObjectDataProps) => {
+  const t = useTranslations('Accordions.Conjunction.object_data');
+
+  const dataArray = [primaryObject, secondaryObject || {} as TypeSatelliteOut];
+
+  return (
+    <div data-pdf={t('title')}>
+      <div className="govuk-body mt-2">
+        {t('space_track')}
+      </div>
+      <div className="overflow-auto">
```

Comment:
Do we need this? Maybe we should add this class to InformationsTable?

Reply:
Good point. Fixed